bin/stafftools

```diff
@@ -37,6 +37,36 @@ class JiraStaffTools < Thor
     output_json(response.body)
   end
 
+  desc "current <git_hub_installation_id>", "Display the current sync item"
+  def current(git_hub_installation_id)
+    jira_host = get_jira_host(git_hub_installation_id)
+    response = client.get(
+      "/api/#{CGI.escape(git_hub_installation_id)}/repoSyncState.json",
+      "jiraHost" => jira_host
+    )
+
+    foundIt = false
+    repos = JSON.parse(response.body)["repos"]
+    doneCount = 0
+
+    repos.each do |repo_id, r|
+      if [r["pullStatus"], r["branchStatus"], r["commitStatus"]] == (["complete"] * 3)
+        doneCount += 1
+        next
+      end
+
+      puts "Working on #{r["repository"]["owner"]["login"]}/#{r["repository"]["name"]}"
+      puts "\t*  pullStatus: #{r["pullStatus"]} @ #{r["lastPullCursor"]}" if r["pullStatus"]
+      puts "\t*  branchStatus: #{r["branchStatus"]} @ #{r["lastBranchCursor"]}" if r["branchStatus"]
+      puts "\t*  commitStatus: #{r["commitStatus"]} @ #{r["lastCommitCursor"]}" if r["commitStatus"]
+      puts "#{repos.length - doneCount} of #{repos.length} Repositories left to sync."
+      foundIt = true
+      break
+    end
```

Comment:
Alternatively, a more ruby way to include index and the found item is:

```suggestion
    found_item = repos.detect.with_index do |(repo_id, r), idx|
      next if [r["pullStatus"], r["branchStatus"], r["commitStatus"]] == (["complete"] * 3)

      puts "Working on #{r["repository"]["owner"]["login"]}/#{r["repository"]["name"]}"
      puts "\t*  pullStatus: #{r["pullStatus"]} @ #{r["lastPullCursor"]}" if r["pullStatus"]
      puts "\t*  branchStatus: #{r["branchStatus"]} @ #{r["lastBranchCursor"]}" if r["branchStatus"]
      puts "\t*  commitStatus: #{r["commitStatus"]} @ #{r["lastCommitCursor"]}" if r["commitStatus"]
      puts "#{repos.length - doneCount} of #{repos.length} Repositories left to sync."
      next r
    end
```

Then below replace `unless foundIt` with `if found_item.nil?`